vscode/scripts/watch.mjs

```diff
@@ -0,0 +1,27 @@
+import esbuild from 'esbuild';
+import config from './esbuild.config.mjs';
+
+function buildClient() {
+  return esbuild.build({
+    ...config,
+    watch: true,
+    entryPoints: ['packages/client/src/index.ts'],
+    outfile: 'dist/index.js',
+  });
+}
+
+function buildServer() {
+  return esbuild.build({
+    ...config,
+    watch: true,
+    entryPoints: ['packages/server/src/index.ts'],
+    outfile: 'dist/server.js',
+  });
+}
+
+async function watch() {
+  await Promise.all([buildClient(), buildServer()]);
+  console.log('👀 Watching for changes...');
```

Comment:
Beginning of something great! This PR was the stepping stone of a DX not just smooth, but friendly and welcoming!